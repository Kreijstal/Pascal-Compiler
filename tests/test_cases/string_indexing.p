program string_indexing;

var
    s: string;
    c: char;
begin
    s := 'Pascal';
    writeln('Initial string is: ', s);
    c := s[1];
    writeln('Read character at index 1: ', c);
    s[4] := 'K';
    c := s[4];
    writeln('Read character at index 4 after modify: ', c);
    writeln('Final string is: ', s);
end.
