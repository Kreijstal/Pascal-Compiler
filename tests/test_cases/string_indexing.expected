Initial string is: Pascal
Read character at index 1: P
Read character at index 4 after modify: K
Final string is: PasKal
